src/main/java/com/artipie/YamlSettings.java

```diff
@@ -45,6 +47,11 @@
  * Settings built from YAML.
  *
  * @since 0.1
+ * @todo #285:30min Settings configuration for GitHub auth.
```

Comment:
I can't understand the description. Can you explain what already done and what is planned to do in the puzzle?